app/controllers/users_controller.rb

```diff
@@ -15,21 +15,35 @@ def edit
 
   def update
     @user = User.find params[:id]
```

Comment:
Ensure that `params[:id]` is the current user signed in (to prevent user from changing other's password). Same thing for `#edit`

Reply:
You probably want `load_and_authorize_resource` from the `cancancan` gem. Then you can define the appropriate abilities in `models/ability.rb`. An admin might want to change other people's passwords.